scripts/nickCrashHelper.js

```diff
@@ -65,25 +65,10 @@ registerPlugin({
 
 
         function nickCheck() {
-            var i = 1,
-                j = 0,
-                nick = sinusbot.getNick(),
-                length = nick.length;
-
-            while (nick.charAt(length - i) == '1') {
-                i++;
-                j++;
-            }
-
-            nick = nick.substring(0, length - j);
-
-            if (j != 0) {
-                debug("Nick set to " + nick + " (found " + j + " crash-numbers after the nick.)");
-            } else {
-                debug("Nick not modified, all seems normal :)");
+            if (sinusbot.getNick() == sinusbot.getActualNick()) { // 0.9.12.2-58b509d or higher needed
```

Comment:
For sure the actual nickname must be different from the real one to set the right nick.

Will correct that small problem, do some tests, and commit new version in about 1hr.